src/App.js

```diff
@@ -1,8 +1,16 @@
+import React, { Component } from 'react';
 import './App.css';
 import Calc from './components/calculator';
 
-function App() {
-  return <Calc />;
+class App extends Component {
```

Comment:
[OPTIONAL] Your all required features are working. However, I would like to leave my suggestion about your styling. 
If I do some calculations in your app. The wireframe is going up and down based on your inputs. Kindly make sure there are no effects on your wireframe :)


https://user-images.githubusercontent.com/69011963/219462228-5536d2f9-844e-4f6f-9273-bfec54e0b3f4.mov

Reply:
- Okay noted thanks. Will work on the optionals.